### ISSUE TITLE
unsupported system: freebsd

### ISSUE THREAD

**Issue by yurivict** [FillZpp/sys-info-rs#35]
```
error: failed to run custom build command for `sys-info v0.5.7`

Caused by:
  process didn't exit successfully: `/usr/home/yuri/rust-learn/tikv/target/debug/build/sys-info-dc048a4e20422394/build-script-build` (exit code: 101)
--- stderr
thread 'main' panicked at 'unsupported system: freebsd', /home/yuri/.cargo/registry/src/github.com-1ecc6299db9ec823/sys-info-0.5.7/build.rs:17:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

warning: build failed, waiting for other jobs to finish...
```

**Comment by FillZpp** [comment]
FreeBSD is not supported yet

**Comment by lwhsu** [comment]
#60 should solve this.

**Comment by lwhsu** [comment]
I think this has been resolved by merging #60 and #61 , and is it possible for us to have a new release? Thanks!

**Comment by FillZpp** [comment]
@yurivict @lwhsu I have published `v0.7.0` which should support freebsd.
